### DISCUSSION TITLE
Is the setting for the color scheme stored locally and it yes, where is it?

### DISCUSSION THREAD

**Discussion by MathisP75** [darkreader/darkreader#11016] [General]
I would like to be able to change the color scheme from catppuccin to nord by editing a file.

**Comment by ngoclong19** [comment]
You can export your settings, edit then import they back again.

Please see here to activate the new user interface, and import and export settings:
`https://darkreader.org/blog/v4-9/`

**Reply by MathisP75** [reply]
Where in my file system do I put the json file to have it apply? I would like to automate it.

**Reply by ngoclong19** [reply]
Can you share what are you trying to achieve?

The extension settings are stored in extension storage, which is browser dependent.
For Firefox:
`https://stackoverflow.com/questions/65448110/where-are-the-data-for-extension-storage-of-a-firefox-extension-stored-in`

**Reply by MathisP75** [reply]
I am on Firefox on Linux and I am making a theme switcher, I would like it to be able to change the dark reader theme as well.

**Reply by MathisP75** [reply]
I searched for the dark reader settings in multiple places on my system, but I did not find anything.

**Reply by ngoclong19** [reply]
Did you try the linked SO answer above. I could locate the .sqlite file in my system.

You can enter `about:profiles` in your browser's address bar to get the location of Firefox profile folder.
To get the extension ID, go here `about:debugging#/runtime/this-firefox`, and look for the UUID of Dark Reader. ~It should be `5396b7fb-2d4e-494c-b730-46d8fe25f4f9`.~

**Reply by MathisP75** [reply]
I had looked at the sqlite file for dark reader but I had not found anything, but since it's compressed according to what you sent me that probably explains it. Thanks for your help, I will look into it.

**Reply by ngoclong19** [reply]
Also linked in the SO thread, this link may come into handy:
`https://www.reddit.com/r/firefox/comments/b5mome/comment/ejexpga/`

**Comment by MathisP75** [comment]
I found it. It's stored in `.mozilla/firefox/<profile-name>/storage-sync-v2.sqlite`
